README.md

```diff
@@ -36,22 +37,38 @@ See the generated [dbt docs site](https://brooklyn-data.github.io/dbt_artifacts/
 ## Quickstart
 
 1. Add this package to your `packages.yml`:
+
 ```
 packages:
   - package: brooklyn-data/dbt_artifacts
     version: 2.3.0
 ```
 
+:contruction-worker: Make sure to fix at least the **minor** version, to avoid issues when a new release is open. See the notes on upgrading below for more detail.
+
 2. Run `dbt deps` to install the package
 
 3. Add an on-run-end hook to your `dbt_project.yml`: `on-run-end: "{{ dbt_artifacts.upload_results(results) }}"`
```

Comment:
Could we add these examples as yml examples like the others as I think it would make it easier to copy and paste for users

Reply:
Good plan - change made.